pkg/network/config/config_test.go

```diff
@@ -756,149 +768,139 @@ service_monitoring_config:
 func TestHTTPNotificationThreshold(t *testing.T) {
 	t.Run("via deprecated YAML", func(t *testing.T) {
 		aconfig.ResetSystemProbeConfig(t)
-		cfg := configurationFromYAML(t, `
-network_config:
-  http_notification_threshold: 100
-`)
-		require.Equal(t, cfg.HTTPNotificationThreshold, int64(100))
+		cfg := configurationFromYAML(t, makeYamlConfigString("network_config", "http_notification_threshold", validNotificationThreshold))
+		require.Equal(t, cfg.HTTPNotificationThreshold, int64(validNotificationThreshold))
 	})
 
 	t.Run("via deprecated ENV variable", func(t *testing.T) {
 		aconfig.ResetSystemProbeConfig(t)
-		t.Setenv("DD_NETWORK_CONFIG_HTTP_NOTIFICATION_THRESHOLD", "100")
+		t.Setenv("DD_NETWORK_CONFIG_HTTP_NOTIFICATION_THRESHOLD", strconv.Itoa(validNotificationThreshold))
 
 		cfg := New()
 
-		require.Equal(t, cfg.HTTPNotificationThreshold, int64(100))
+		require.Equal(t, cfg.HTTPNotificationThreshold, int64(validNotificationThreshold))
 	})
 
 	t.Run("via YAML", func(t *testing.T) {
 		aconfig.ResetSystemProbeConfig(t)
-		cfg := configurationFromYAML(t, `
-service_monitoring_config:
-  http_notification_threshold: 100
-`)
-		require.Equal(t, cfg.HTTPNotificationThreshold, int64(100))
+		cfg := configurationFromYAML(t, makeYamlConfigString("network_config", "http_notification_threshold", validNotificationThreshold))
```

Comment:
This was `service_monitoring_config` before.

Reply:
Indeed it was. 🙇